src/lang/lang_types.ml

```diff
@@ -967,7 +964,7 @@ let rec ( <: ) a b =
               let (o, lbl, t'), l2' = get_param [] l2 in
               (* Check on-the-fly that the types match. *)
               begin
-                try t <: t'
```

Comment:
This seems unrelated to the PR?

Reply:
This one is completely related to the PR: the variance was wrong, and it seems that this is the first time that we are really using subtyping for function arguments (to discard methods)...